pkgs/tools/networking/asynk/default.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv, fetchurl, python2, python2Packages, makeWrapper }:
+
+stdenv.mkDerivation rec {
+  version = "1.1.0";
+  name = "ASynK-${version}";
+
+  src = fetchurl {
+    name = "${name}.tar.gz";
+    url = "https://github.com/skarra/ASynK/archive/v${version}.tar.gz";
+    sha256 = "0sh2bxda4pi4h2l51zgsbq1ap2ahlvgjcfl6fpzrlbbjffbg8qjb";
+  };
+
+  propagatedBuildInputs = [ python2 makeWrapper python2Packages.tornado python2Packages.requests python2Packages.dateutil ];
```

Comment:
You can also use: `propagatedBuildInputs = with python2Packages; [ python2 makeWrapper tornado requests dateutil ];` to avoid some typing.